terraform/hcl/applicable/req_sw_terraform_aws_athena_encrypt.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_athena_database" "vulnerable_example" {
+    name = "dummy"
+    # encryption_configuration is not set
+}
```

Comment:
[comment]: <> (FrogbotReview Iac 1 1 4 2 vulnerable_example /terraform/hcl/applicable/req_sw_terraform_aws_athena_encrypt.tf)

### 🛠️ Infrastructure as Code Vulnerability
	
| Severity | Finding |
| :---: | :---: |
| ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableMediumSeverity.png)<br> | AWS Athena database does not usedata-at-rest encryption |

<details>
<summary> <b>Full description</b> </summary>
<br>

Resource `aws_athena_database` should have key `encryption_configuration`

Vulnerable example - 
```
resource "aws_athena_database" "vulnerable_example" {
      # encryption_configuration is not set
}
```

Secure example -
```
resource "aws_athena_database" "secure_example" {
      encryption_configuration {
        encryption_option = "SSE_KMS"
        kms_key      = aws_kms_key.example.arn
      }
}
```

</details>


---
<div align="center">

[🐸 JFrog Frogbot](https://github.com/jfrog/frogbot#readme)

</div>